stable/gcpdisk-csi-driver/templates/csi-gcpdisk-controller.yaml

```diff
@@ -0,0 +1,116 @@
+kind: StatefulSet
+apiVersion: apps/v1
+metadata:
+  name: csi-gce-pd-controller
+  namespace: {{ .Release.Namespace }}
+{{ include "gcpdisk.labels" . | indent 2 }}
+spec:
+  serviceName: "csi-gce-pd"
+  replicas: 1
+  selector:
+    matchLabels:
+      app: gcp-compute-persistent-disk-csi-driver
+  template:
+    metadata:
+{{ include "gcpdisk.labels" . | indent 6 }}
+        app: gcp-compute-persistent-disk-csi-driver
+    spec:
+      # Host network must be used for interaction with Workload Identity in GKE
+      # since it replaces GCE Metadata Server with GKE Metadata Server. Remove
+      # this requirement when issue is resolved and before any exposure of
+      # metrics ports
+      hostNetwork: true
+      serviceAccountName: csi-gce-pd-controller-sa
+      nodeSelector:
+        beta.kubernetes.io/os: linux
+      priorityClassName: system-cluster-critical
+      containers:
+        - name: csi-provisioner
+          image: quay.io/k8scsi/csi-provisioner:v1.4.0
+          args:
+            - --csi-address=$(ADDRESS)
+            - --timeout=30s
```

Comment:
perhaps this should be configurable :/  I can imagine it failing under certain load

Reply:
the timeout?

Reply:
and if so, it would retry ;)

Reply:
I meant to have an args property in values.yaml where you have these settings. Perhaps in a bigger cluster they need to be customize due to higher timeouts. I can do it in a follow-up pr

Reply:
I can do it as well, this is not the point ;)
but we do not need it right now to get it running